sde_sampler/losses/oc.py

```diff
@@ -36,12 +37,15 @@ def __init__(
         self.max_rnd = max_rnd
 
         # SDE controls
-        self.sde_ctrl_noise = sde_ctrl_noise
-        self.sde_ctrl_dropout = sde_ctrl_dropout
-        if self.method in ["kl", "kl_ito"]:
-            for attr in ["sde_ctrl_noise", "sde_ctrl_dropout"]:
-                if getattr(self, attr) is not None:
-                    logging.warning("%s should only be used for the log-variance loss.")
+        if self.sde is not None:
```

Comment:
We can remove this whole block until l48 and raise a ValueError if method is not in `["kl", "kl_ito"]`.